pom.xml

```diff
@@ -25,7 +25,7 @@ when an elasticsearch client starts.
         <elasticsearch.version>6.1.3</elasticsearch.version>
         <test.elasticsearch.version>${elasticsearch.version}</test.elasticsearch.version>
         <slf4j.version>1.7.13</slf4j.version>
-        <log4j.version>2.8.1</log4j.version>
+        <log4j.version>2.10.0</log4j.version>
```

Comment:
Could you make that change in another PR?

Reply:
Yeah. Would you like any other version updates addressed there? I mainly knew about that one since I'm one of their maintainers.

Reply:
I generally prefer updating librairies one by one (one per PR) so it's easier to track the changes even if it's super small.